analysis/topEFT/check_yields.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+
+# What we want to call the output files
+FILE_NAME="check_yields"
```

Comment:
Is this meant to be the prototype for the script that would run automatically?  If so, I'd set it up to run over the same data sample that the reference json is produced from to generate the json instead of assuming that's already been done.

Reply:
Yes, this is meant to be the prototype for the script that will be run automatically, and yes, this script runs the processor over the data sample used to produce the reference json, as you suggested.